manual/manual.tex

```diff
@@ -82,18 +74,18 @@
 
 \newcommand{\version}{\input{VERSION}}
 
+\newcommand{\rtm}{\textsuperscript{\textregistered}}
+
 % Read information about the repository.
 \input{env}
 
-% Don't indent paragraphs; instead, just leave some vertical space.
-\parindent 0pt\parskip 6pt
-
 \begin{document}
 
-  \title{seL4 Reference Manual\\Version \version}
+  \title{seL4\\ Reference Manual}
+  \subtitle{Version \version}
 
   \author{seL4 Foundation}
-  \AuthorEmail{https://sel4.systems}
+  \email{https://sel4.systems}
```

Comment:
`\url{}` ?

Reply:
The `\email` command is from the `sel4.sty` package. Usually you want an email there, but it just renders it as url anyway, so I think that's fine.